src/toil/jobStores/aws/jobStore.py

```diff
@@ -742,6 +742,13 @@ def bucket_creation_pending(error):
                             # NoSuchBucket. We let that kick us back up to the
                             # main retry loop.
                             assert self.getBucketRegion(bucket_name) == self.region
+
+                            owner_tag = os.environ.get('SET_OWNER_TAG')
```

Comment:
Shouldn't this be something like `TOIL_AWS_OWNER_TAG`? We probably are going to want to recommend that users set the environment variable when they run Toil workflows, and not just for the tests.

If we were really cool we would default to the IAM user we are logging in as from the AWS config, but that might be a new ticket.

Reply:
We also probably want to list it in the environment variables documentation file.

Reply:
Hmmm... I envisioned using this on Google as well, since we'll probably want to control resources there too eventually.  `TOIL_OWNER_TAG` makes sense to me, but maybe I should make something more generic that will simply add any number of tags specified to any bucket we create.

Reply:
I think we only really need one tag for now. If we want more later we could extend it to look for `TOIL_XXXXX_TAG` and apply all of them.